src/context/timestampsContext.js

```diff
@@ -0,0 +1,56 @@
+import { createContext, useContext, useEffect, useMemo, useState } from 'react';
+import PropTypes from 'prop-types';
+import useFetchTs from 'hooks/useFetchTs';
+import { CSV_URL } from 'const';
+
+const TimestampsContext = createContext();
+
+export const TimestampsConsumer = TimestampsContext.Consumer;
+
+const TimestampsProvider = function TimestampsProvider({ children }) {
+  const [drTs, setDrTs] = useState(null);
+  const [instTs, setInstTs] = useState(null);
+
+  const drTsRequest = useFetchTs(CSV_URL.DOCTORS_TS);
+  const instTsRequest = useFetchTs(CSV_URL.INSTITUTIONS_TS);
```

Comment:
I would never request `institutions.csv.timestamp` - no need as we only need one timestamp

Reply:
done.